src/test/generate-sym-test.py

```diff
@@ -66,7 +66,7 @@ def process_source_file(file):
 }, symbols_from_source[] = {''')
 
 for dirpath, _, filenames in sorted(os.walk(sys.argv[2])):
-    for filename in filenames:
+    for filename in sorted(filenames):
```

Comment:
Shouldn't the "sorted()" above make this already redundant? I have seen the exact same issue in Debian's repro tests, have you confirmed this fixed the issue? If so that would be great

Reply:
Yes, I tested both with and without the patch and the patch helps to avoid variations in `/usr/lib/systemd/tests/unit-tests/test-libsystemd-sym` and `/usr/lib/systemd/tests/unit-tests/test-libudev-sym` . It seems, the `sorted` above sorts the directories (stored in `_`).

Reply:
Ah I completely missed that and spent a lot of time scratching my head, excellent find